src/js/models/layer.js

```diff
@@ -134,12 +134,12 @@ define(function (require, exports, module) {
         /**
          * @type {Immutable.List.<Shadow>}
          */
```

Comment:
Why the change from null to `Immutable.List()`? We have other Lists in this record that initialize as null.